src/components/LoginComponent/LoginComponent.jsx

```diff
@@ -1,5 +1,85 @@
-import React from 'react';
+import React, { useState } from "react";
+import "./logincomponent.scss";
+import { useHistory } from "react-router-dom";
 
 export default function LoginComponent() {
-  return <div></div>;
+  const [userName, setUserName] = useState("");
+  const [password, setPassword] = useState("");
+
+  const history = useHistory();
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    let information = {
+      userName: userName,
+      password: password,
+    };
+
+    const requestOptions = {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(information),
+    };
+    fetch("https://stingy-cherry-sight.glitch.me/login", requestOptions)
+      .then((res) => res.json())
+      .then((res) => {
+        console.log(res);
+        if (res.success === true) {
+          localStorage.setItem("token", res.token); //storing the token in localStorage.
+          history.push("/student-tracker");
```

Comment:
This route should go to /choose-class not /student-tracker

Reply:
I only used it for a test but i have now changed it